pkg/kubectl/cmd/util/openapi/validation/validation.go

```diff
@@ -61,15 +61,15 @@ func (v *SchemaValidation) validateList(object interface{}) []error {
 		return []error{errors.New("invalid object to validate")}
 	}
 
-	errs := []error{}
+	err := []error{}
```

Comment:
I think this name could be `allErrors` `errors` or `errList`... also please squash your commit